README.md

```diff
@@ -7,3 +7,23 @@ Find out more at the
 Production version is stored [here](https://blockly-demo.appspot.com/static/demos/blockfactory/index.html) ([source](https://github.com/google/blockly/tree/master/demos/blockfactory)).
 
 Want to contribute? Great! First, read [our guidelines for contributors](https://developers.google.com/blockly/guides/modify/contributing).
+
+## Setting Up
+
+### Install node and NW.js
+
+In order to properly run and test Blockly's devtools, install [node.js](https://nodejs.org/en/download/) and [NW.js](https://nwjs.io/downloads/) on your computer. Instructions for installation can be found on the linked sites.
```

Comment:
nit: We generally format lines to be < 100 chars except where a link makes it impossible.